src/util/logger.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// N.B! don't import this on the client!
+
+const bunyan = require('bunyan');
+require('source-map-support').install();
+
+let log;
+
+if (!log) {
+  log = bunyan.createLogger({ name: 'ndla-frontend' });
```

Comment:
Kanskje ha listing-frontend her? Brukes denne forresten?
```suggestion
  log = bunyan.createLogger({ name: 'listing-frontend' });
```

Reply:
Jeg så at ndla-frontend håndterte 500 errors på denne måten. Trodde at siden det gjøres slik der så var det en ordentlig måte å gjøre ting på. Kanskje det er bedre å bare `console.error(error)` i server.js enn å bruke det fancy greier her?